### DISCUSSION TITLE
Forking not working

### DISCUSSION THREAD

**Discussion by viraj-s15** [community/community#47134] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I try to fork any application, I am send to the page with the unicorn that says "This page is taking too long to load." This happens when trying to fork any repo. Can someone help me out with this?

**Comment by Sergei190** [comment]
Здравствуйте. Почему форк не  работает?

**Reply by viraj-s15** [reply]
I dont know why its happening. I gave all the information I had. Just wanted to know whether this is something I am doing wrong / something happening with me only

**Comment by Bejte** [comment]
same thing is happening to me

**Comment by medAmineRg** [comment]
same thing here

**Comment by inesartero95** [comment]
Same here

**Comment by Rickos99** [comment]
Same here.

On the first attempt, I get (after some time) a message that "This page is taking too long to load".

![image](https://user-images.githubusercontent.com/12218241/218310707-ac812c8c-8aed-499a-a4b0-520582fc3e65.png)

On the second attempt, I get a 500 message. 

![image](https://user-images.githubusercontent.com/12218241/218310687-bc45eb24-a05a-49ef-a5ba-ce51c0b9405a.png)

## Things tried
I have tried to:
- exit the browser and then open it again,
- fork the repo in another browser,
- sign out and then sign in to GitHub again.
